go/config/config_test.go

```diff
@@ -0,0 +1,301 @@
+// Copyright 2019 The Grafeas Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func defaultServerConfig(t *testing.T) *ServerConfig {
```

Comment:
Is the `t` param necessary here? Also, could you extract this and the below into vars directly?

Reply:
I did have these as vars initially, but @vtsao requested I use helper functions for consistency and for all testing helper functions to take in (t *testing.T) so that they can be marked as helpers with `t.Helper()`, see his comments below:

https://github.com/grafeas/grafeas/pull/383#discussion_r327726476
https://github.com/grafeas/grafeas/pull/383#discussion_r327726954

Reply:
Sorry for the confusion from 2 reviewers! Let me sync up with @vtsao offline and get back to you on this.

Reply:
Yes, please call t.Helper() for to mark helper functions as a best practice in testing. See:
https://golang.org/pkg/testing/#T.Helper

generally it's used to attribute failures in the test helper to the line where the helper is called so you don't obscure the test logs. I know there's no possibility of failure in this testing function, but still good to call as a best practice.

Reply:
Amended.